dspace-server-webapp/src/test/java/org/dspace/app/rest/ProcessRestRepositoryIT.java

```diff
@@ -43,6 +43,13 @@
 
     @Before
     public void setup() throws SQLException {
+        CollectionUtils.emptyIfNull(processService.findAll(context)).stream().forEach(process -> {
+            try {
+                processService.delete(context, process);
+            } catch (SQLException e) {
+                throw new RuntimeException(e);
+            }
+        });
```

Comment:
This looks like unrelated code.  I don't understand what this would have to do with user registration / forgot password.  Was this accidentally included in a recent commit?

Reply:
This has been removed